step.sh

```diff
@@ -34,7 +33,14 @@ case $lint_range in
   git diff --name-only | while read filename; do run_swiftlint "${filename}" "${reporter}" "${FLAGS}"; done
   git diff --cached --name-only | while read filename; do run_swiftlint "${filename}" "${FLAGS}"; done 
   ;;
-"all") 
-  swiftlint lint --reporter "${reporter}" ${FLAGS}
+"all")
+  output="$(swiftlint lint --reporter "${reporter}" ${FLAGS})"
+  envman add --key "SWIFTLINT_REPORT" --value "${output}"
+  echo "Saved swiftlint output in SWIFTLINT_REPORT"
+  filename="swiftlint_report.${reporter}"
```

Comment:
This will not work correctly I think. We cannot have a file called `.xcode` etc 😅

Maybe this could help?
https://github.com/limamedeiros/bitrise-step-swiftlint/blob/master/step.sh#L19

Reply:
This helps a lot, thanks! I will update this part soon.